CleverTapSDK/InApps/Matchers/CTTriggersMatcher.m

```diff
@@ -124,9 +135,8 @@ - (BOOL)matchFirstTimeOnly:(CTEventAdapter *)event trigger:(CTTriggerAdapter *)t
     if (!trigger.firstTimeOnly) {
```

Comment:
Do we need `(CTEventAdapter *)event` argument in this method?

Reply:
resolved.